src/pcre2_compile.c

```diff
@@ -782,12 +782,16 @@ static const uint8_t opcode_possessify[] = {
   OP_CRPOSRANGE, 0,        /* CRRANGE, CRMINRANGE */
   0, 0, 0, 0,              /* CRPOS{STAR,PLUS,QUERY,RANGE} */
 
-  0, 0, 0,                 /* CLASS, NCLASS, XCLASS */
+  0, 0, 0, 0,              /* CLASS, NCLASS, XCLASS, ECLASS */
   0, 0,                    /* REF, REFI */
   0, 0,                    /* DNREF, DNREFI */
-  0, 0                     /* RECURSE, CALLOUT */
+  0, 0,                    /* RECURSE, CALLOUT */
 };
 
+/* Compile-time check that the table has the correct size. */
+typedef int check_opcode_possessify[
```

Comment:
Maybe we should introduce static asserts.

Reply:
This _is_ a static assert. I guess you mean, you'd prefer to wrap it in a macro? We could do that.